auacm/app/modules/submission_manager/views.py

```diff
@@ -35,7 +35,7 @@ def submit():
 
     attempt = models.Submission(
         username=current_user.username,
-        pid=request.form['pid'].lower(),
+        pid=request.form['pid'],
```

Comment:
Agreed. We need to make a few modifications to the database and make some rules for how columns need to be laid out.